app/controllers/index.js

```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
     return this.filteredEvents ? this.filteredEvents.filter(event => {return event.isFeatured}) : null;
 
   }),
+    marker: {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)